src/private/templates/index.latte

```diff
@@ -31,7 +31,12 @@
                             {$news["title"]}
                         {/ifset}
                     </h5>
-                    <div>{strtotime($news["added"])|fuzzyDateAbbr}</div>
+                    <div>
+                        {strtotime($news["added"])|fuzzyDateAbbr}
+                        {if $news["edited"] !== null}
+					        <i class="fas fa-pencil-alt mr-0 ml-1 news-edited" data-toggle="tooltip" n:attr="data-date => $news['edited']" data-html="true"></i>
```

Comment:
1. What's the purpose of the `news-edited` class?
2. Why is the HTML allowed in the tooltip?

Reply:
1. The `news-edited` class is to make javascript recognize the element.
2. Deleted.